src/ReflectionFunction.php

```diff
@@ -136,6 +161,17 @@ public function __toString()
      */
     protected function __initialize()
     {
-        parent::__construct($this->getName());
+        parent::__construct($this->functionName);
+    }
+
+    /**
+     * Has class been loaded by PHP.
+     *
+     * @return bool
+     *     If class file was included.
+     */
+    public function wasIncluded()
+    {
+        return function_exists($this->functionName);
```

Comment:
Theoretically this is possible, but do we handle case with functions auto-loaded (if that's even possible)?

Reply:
To my knowledge, PHP function autoloading has been proposed several times, but never included in any mainstream branch. If function auto loading were implemented, and `function_exists()` had the same semantics as `class_exists()` we would need to add a `false` as a second parameter, but that's simply conjecture at this point.